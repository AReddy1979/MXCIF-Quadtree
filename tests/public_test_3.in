INIT_QUADTREE(5)
HORIZ_NEIGHBOR(7CP8)
SEARCH_POINT(63065,2515)
DISPLAY()
VERT_NEIGHBOR(FK)
INIT_QUADTREE(5)
NEAREST_RECTANGLE(54197,7895)
HORIZ_NEIGHBOR(X08YDL)
TRACE ON
TRACE ON
DELETE_POINT(6889,53332)
INSERT(40127)
SEARCH_POINT(64640,12392)
LIST_RECTANGLES()
VERT_NEIGHBOR(Y)
TRACE OFF
INSERT(45708)
INIT_QUADTREE(6)
INIT_QUADTREE(8)
HORIZ_NEIGHBOR(U)
CREATE_RECTANGLE(OKNW,49875,2017,60923,260)
SEARCH_POINT(7659,38155)
DISPLAY()
DELETE_RECTANGLE(5PMQ4L)
TRACE OFF
