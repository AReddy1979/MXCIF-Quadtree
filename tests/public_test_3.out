1A 5
12A 7CP8
5 63065 2515
2
12B FK
1A 5
13 54197 7895
12A X08YDL
8B 6889 53332
7 40127
5 64640 12392
3
12B Y
7 45708
1A 6
1A 8
12A U
4 OKNW 49875 2017 60923 260
5 7659 38155
2
8A 5PMQ4L
